client/src/components/DatePicker/DatePicker.stories.js

```diff
@@ -11,4 +11,4 @@ function handleChange(e) {
 
 
 storiesOf('DatePicker', module)
-    .add('DatePicker', () => <DatePicker label={"Requested Date*"} name={"Requested Date"} value={"November 11"} min={"February 29, 2020"} step={"1"} max={"February 28, 2022") onChange={handleChange}/>)
+    .add('DatePicker', () => <DatePicker label={"Requested Date*"} name={"Requested Date"} value={"November 11"} min={"February 29, 2020"} step={"1"} max={"February 28, 2022"} onChange={handleChange}/>)
```

Comment:
This code is causing issues in your code base. For now comment it out until the person who is working on it fixes the bugs it introduces.